common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py

```diff
@@ -36,30 +36,26 @@ def handle(self, *args, **options):
-
-                    course_chapters = modulestore().get_items(
-                        course.id,
-                        qualifiers={'category': 'course'}
-                    )
-                    for index_chapter, chapter in enumerate(course_chapters[0].children):
-                        if max_module < current_module:
-                            sequentials = modulestore().get_item(chapter)
-                            for index_sequential, sequential in enumerate(sequentials.children):
-                                verticals = modulestore().get_item(sequential)
-                                for index_vertical, vertical in enumerate(verticals.children):
-                                    vertical_blocks = modulestore().get_item(vertical)
-                                    for index_block, block in enumerate(vertical_blocks.children):
-                                        if (block.block_type == ORA_BLOCK_TYPE and
-                                                can_auto_score_ora(enrollment, course, block, index_chapter)):
-                                            anonymous_user = get_anonymous_user(user, course.id)
-                                            student = {
-                                                'anonymous_user_id': anonymous_user.anonymous_user_id
-                                            }
-                                            autoscore_ora(course.id, unicode(block), student)
-
-                        max_module = max_module + 1
+                max_module = 1
+
+                course_chapters = modulestore().get_items(
+                    course.id,
+                    qualifiers={'category': 'course'}
+                )
+                for index_chapter, chapter in enumerate(course_chapters[0].children):
+                    if max_module < current_module:
+                        sequentials = modulestore().get_item(chapter)
+                        for index_sequential, sequential in enumerate(sequentials.children):
+                            verticals = modulestore().get_item(sequential)
+                            for index_vertical, vertical in enumerate(verticals.children):
+                                vertical_blocks = modulestore().get_item(vertical)
+                                for index_block, block in enumerate(vertical_blocks.children):
+                                    if (block.block_type == ORA_BLOCK_TYPE and
+                                            can_auto_score_ora(enrollment, course, block, index_chapter)):
+                                        anonymous_user = get_anonymous_user(user, course.id)
+                                        student = {
+                                            'anonymous_user_id': anonymous_user.anonymous_user_id
+                                        }
+                                        autoscore_ora(course.id, unicode(block), student)
+
+                    max_module = max_module + 1
```

Comment:
It is just a suggestion and totally on your call, 
We can decrease indent level by inverting the condition like 
```
                    if max_module >= current_module:
                        max_module = max_module + 1
                        continue

                    sequentials = modulestore().get_item(chapter)
                    for index_sequential, sequential in enumerate(sequentials.children):
                        verticals = modulestore().get_item(sequential)
                        for index_vertical, vertical in enumerate(verticals.children):
                            vertical_blocks = modulestore().get_item(vertical)
                            for index_block, block in enumerate(vertical_blocks.children):
                                if (block.block_type == ORA_BLOCK_TYPE and
                                        can_auto_score_ora(enrollment, course, block, index_chapter)):
                                    anonymous_user = get_anonymous_user(user, course.id)
                                    student = {
                                        'anonymous_user_id': anonymous_user.anonymous_user_id
                                    }
                                    autoscore_ora(course.id, unicode(block), student)
```

Reply:
This command will have major refactoring soon, in another story. Probably this whole code will shrink to one or two lines.